### ISSUE TITLE
mpv-osc-thumbfast-git: osc not installed

### ISSUE THREAD

**Issue by lilydjwg** [archlinuxcn/repo#3539]
### 问题类型 / Type of issues

* 打包错误 / packaging error

### 受影响的软件包 / Affected packages

* mpv-osc-thumbfast-git

----

[Upstream README](https://github.com/po5/thumbfast#ui-support) provides several choices for the UI, but none is installed or depended by this package. No UI is available after installing this package.

**Comment by zjuyk** [comment]
Yes, the osc choices should be optdepends. Package them and add into this PKGBUILD later.

**Comment by zjuyk** [comment]
Actually, I involve this package because I was using mpv-osc-modern-git. So I add mpv-osc-thumbfast-git as depends of modern. 

Yes, it should be better to treat the UI support packages as optdepends of thumbfast.

- [ ] remove thumbfast depends in mpv-osc-modern-git
- [ ] package other UI support packages
  - [ ] mpv-osc-uosc-git
  - [ ] mpv-osc-modernx-git
  - [ ] mpv-osc-progressbar-git
  - [ ] mpv-osc-tethys-git
  - [ ] mpv-osc-oscc-git
  - [ ] mpv-osc-mfpbar-git
